composer.json

```diff
@@ -29,6 +29,9 @@
         "symfony/swiftmailer-bundle": "^3.1",
         "symfony/twig-bundle": "4.4.*",
         "symfony/yaml": "4.4.*",
+        "toiba/fullcalendar-bundle": "^6.2",
```

Comment:
Ce paquet est déprécié: https://packagist.org/packages/toiba/fullcalendar-bundle

Ils suggèrent d'utiliser https://github.com/tattali/CalendarBundle. Est-ce que ça marcherait de votre coté ?

Reply:
C’est noté. J’ai pris le nouveau package maintenu. En plus ça servira pour link avec l’api de Google.